Cargo.toml

```
@@ -25,6 +25,10 @@ name = "rustzcash"
 path = "src/rust/src/rustzcash.rs"
 crate-type = ["staticlib"]
 
+[[bin]]
+name = "zcashd-wallet-tool"
+path = "src/rust/src/wallet_tool.rs"
```

This is a confusing place to locate the Rust binaries, because it makes the module accessible as part of the library too. Move this instead to:
```suggestion
path = "src/rust/bin/wallet_tool.rs"
```
(which more closely matches a normal Rust package, which would have `src/lib.rs` and `src/bin/foobar.rs`)